python/tvm/relay/op/qnn/qnn.py

```diff
@@ -0,0 +1,74 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#pylint: disable=invalid-name, too-many-lines
+"""Neural network operations."""
+from __future__ import absolute_import as _abs
+from . import _make
+
+def requantize(input_data, input_zero_point, input_scale, output_zero_point,
+               output_scale, out_dtype="int32", use_int_compute=False,
+               rounding_mode="FE_UPWARD"):
```

Comment:
TFLite requires "round_awar_from_zero". Could you explain why we set it "FE_UPWARD" be the default value? Because of MXNet's requirement?

Reply:
It can be "round_away_from_zero". These two rounding modes provide two points in the performance accuracy tradeoff curve. FE_UPWARD has fewer operators compared to round_way_from_zero. But, I think you are right, we should set "round_away_from_zero" by default. If somebody wants better performance, they can set it to FE_UPWARD using a network-level config.